conllulex/conllulex_to_json.py

```diff
@@ -502,6 +514,8 @@ def _validate_sentences(corpus, sentences, errors, validate_upos_lextag, validat
                 elif ss is None:
                     assert_(False, f"Missing supersense annotation in lexical entry: {lex_expr}", token=lex_expr)
                 elif ss not in valid_ss:
+                    if sent_id == 'lp_hi_26-73':
+                        print ('here')
                     if lexcat not in lang_config['lexcat_exception_list']:
```

Comment:
looks like a debug print got left in by accident?

Reply:
Yeah, removed it in this [commit](https://github.com/nert-nlp/conllulex/pull/5/commits/7164097acfd100b4b0b22e5d89cdf16dfc558bc5):